dpctl/tensor/libtensor/include/kernels/elementwise_functions/imag.hpp

```diff
@@ -70,11 +71,13 @@ template <typename argT, typename resT> struct ImagFunctor
     resT operator()(const argT &in) const
     {
         if constexpr (is_complex<argT>::value) {
```

Comment:
There is `is_complex_v` available in the `type_utils.hpp`
```suggestion
        if constexpr (is_complex_v<argT>) {
```

Reply:
yes, did not exist at the time these functions were made. I can switch to it.